IoTConnect_2.0/IoTConnect_2.0/IoTConnectManager/IoTConnectManager.swift

```diff
@@ -209,18 +227,19 @@ class IoTConnectManager {
             let metaInfo = self.dictSyncResponse["meta"] as? [String:Any]
             df = metaInfo?["df"] as? Int ?? 0
             
-//            if let time = prevSendDataTime{
-//                let diff = Int(Date().timeIntervalSince(time))
-//                print("df \(df) diff\(diff) \(time) \(Date())")
-//                if diff >= df{
-//                    validateData(data: data)
-//                }else{
-//                    print("Diff is lt")
-//                }
-//            }else{
-//                print("prevsendData is nil")
+            //check for data frequency
+            if let time = prevSendDataTime{
+                let diff = Int(Date().timeIntervalSince(time))
+                print("df \(df) diff\(diff) \(time) \(Date())")
+                if diff >= df{
+                    validateData(data: data)
+                }else{
+                    print("Diff is lt")
```

Comment:
What happens if it goes in else case?

Reply:
If difference between time is less then df value from portal then nothing will happen means data is not going to be sent on MQTT.